windows-image-install.ps1

```diff
@@ -0,0 +1,50 @@
+# Following script works with the Windows 2016 image provided by GCE. If running a different version of Windows/Powershell, changes may be needed
```

Comment:
Suggest adding a line break to this comment to enhance readability.

Reply:
done